modules/vulcan/src/main/scala/fs2/kafka/vulcan/AvroSettings.scala

```diff
@@ -184,7 +206,7 @@ object AvroSettings {
 
   private[this] def create[F[_]](
     schemaRegistryClient: F[SchemaRegistryClient]
-  )(implicit F: Sync[F]): AvroSettings[F] =
+  )(implicit F: Async[F]): AvroSettings[F] =
```

Comment:
We need `Async` here now because the default retry needs `Async`. If this sounds problematic (noting this is effectively the only constructor) we could provide something like a new `createNoRetry[F[_]: Sync]` - let me know what you think.